packages/api-bindings/src/graphql/publications.graphql

```diff
@@ -48,3 +48,45 @@ query Publications(
     }
   }
 }
+
+query ExplorePublications(
+  $limit: LimitScalar!
+  $cursor: Cursor
+  $timestamp: TimestampScalar
+  $sortCriteria: PublicationSortCriteria!
+  $sources: [Sources!]
+  $publicationTypes: [PublicationTypes!]
+  $noRandomize: Boolean
+  $excludeProfileIds: [ProfileId!]
+  $observerId: ProfileId
+) {
+  result: explorePublications(
+    request: {
+      limit: $limit
+      cursor: $cursor
+      timestamp: $timestamp
+      sortCriteria: $sortCriteria
+      sources: $sources
+      publicationTypes: $publicationTypes
+      noRandomize: $noRandomize
+      excludeProfileIds: $excludeProfileIds
```

Comment:
To minimize TS type def and GQL definitions we can simply pass the entire request object and have the wrapping hook be the sole place of indirection between outer facing parameters and GQL variables.